gen/schema-header.act

```diff
@@ -1590,9 +1590,13 @@ def yang_leaf_to_acton_arg_type(leaf: DNodeLeaf, loose: bool=False) -> str:
 def yang_leaf_to_getval(leaf: DNodeLeaf) -> str:
     # TODO: uh, do we need loose arg?
     optional = is_optional_yang_leaf(leaf)
-    optional_str = "opt_" if optional else ""
-    t = yang_type_to_acton_type(leaf.type_)
-    return optional_str + t
+    getval_str = "opt_" if optional else ""
```

Comment:
I see how this doesn't fit into yang_type_to_acton_type since bits is not an acton type. This is about getting the name of the getter. We have the yang_leaf_to_getval - shouldn't it go in there somewhere?

Reply:
Did you mean some other method than yang_leaf_to_getval?

Reply:
Hehe, no, I think I meant `yang_leaf_to_getval` - I mean to place the logic for computing the name of the deserializing getter-function in `yang_leaf_to_getval`.

The goal here is to get the type of the deserialize / getter function., right!? We used to compute the name via `yang_type_to_acton_type` which worked since all the yang types up to that point had a simple mapping to acton types where the name of the acton type and the getter function name would correspond, like `str` / `get_str`. That is no longer the case for bits, which we said we want to store as a `set[str]` but the getter is `get_bits` or similar.

I think the purpose for the `yang_leaf_to_getval` was precisely to return the function name for the deserialize / get-value function, i.e. what we are trying to do here. So instead of adding extra logic inline here, we should move it to `yang_leaf_to_getval` and call that it. Does that make sense?